metrics/core/src/main/java/org/hyperledger/besu/metrics/BesuMetricCategory.java

```diff
@@ -40,9 +42,11 @@
   public static final Set<MetricCategory> DEFAULT_METRIC_CATEGORIES;
 
   static {
-    // Why not ROCKSDB and KVSTORE_ROCKSDB_STATS?  They hurt performance under load.
+    // Why not ROCKSDB and KVSTORE_ROCKSDB_STATS, KVSTORE_PRIVATE_ROCKSDB_STATS?  They hurt
+    // performance under load.
     final EnumSet<BesuMetricCategory> besuCategories =
-        EnumSet.complementOf(EnumSet.of(KVSTORE_ROCKSDB, KVSTORE_ROCKSDB_STATS));
+        EnumSet.complementOf(
```

Comment:
Why is `KVSTORE_PRIVATE_ROCKSDB` not needed in the set but `KVSTORE_ROCKSDB` is?

Reply:
yeah ... it probably is :) nice.

Reply:
done.